src/Connection/ConnectionInterface.php

```diff
@@ -25,6 +25,8 @@
  *
  * It allows you to access and manipulate databases in a database-agnostic way, so you can write code that works with
  * different database systems without having to worry about the specific details of each one.
+ *
+ * @psalm-type ParamsType = array<string,mixed>|list<mixed>
```

Comment:
```suggestion
 * @psalm-type ParamsType = array<non-empty-string, mixed>|list<mixed>
```
Name of a parameter should be non-empty